hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -152,6 +152,10 @@ public void notifyLeaderChanged(RaftGroupMemberId groupMemberId,
 
     LOG.info("current SCM becomes leader of term {}.", term);
     scm.getScmContext().updateIsLeaderAndTerm(true, term);
+    if (scm.getScmContext().isLeader()) {
```

Comment:
Why we need this isLeader check ?

Reply:
you are right, we do not need this check.